lib/Service/FilesService.php

```diff
@@ -107,7 +107,7 @@ public function fillRestoringData(RestoringData $data, string $path): void {
 		if (!is_dir($data->getAbsolutePath() . rtrim($path, '/'))) {
 			if ($data->getType() === RestoringData::ROOT_DATA
 				&& !$this->configService->getAppValueBool(ConfigService::INCLUDE_LOGS)
-				&& $path === 'nextcloud.log') {
+				&& substr($path, 0, 13) === 'nextcloud.log') {
```

Comment:
Might not be compatible with custom log file

Reply:
I now get value of `logfile`